### PR TITLE
Add Korean translations

### PR DESCRIPTION
**Pull request by jhk1090**
PR with file README_Korean.md and i18n.js modification
I hope you accept my PR!

### CODE REVIEW

File: auto_py_to_exe/web/js/i18n.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ const translationMap = {
                 fi: 'Skriptin sijainti',
                 vi: 'Vị trí tập lệnh',
                 nl: 'Script Locatie',
+                ko: "스크립트 위치"
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:35]: Can you add commas to the end of these new additions? This helps reduce the diff in future changes.

```suggestion
                ko: "스크립트 위치",
```

(more below)

File: README-Korean.md
Comment (brentvollebregt) [inline README-Korean.md:1]: Add a link to this file in `README.md` where the others are 😃

**Review by brentvollebregt — CHANGES_REQUESTED** [review]
Looking good 👍 Just a couple of requests

**Comment by jhk1090** [comment]
I've reflected your feedback!
Check out my last commit

**Review by brentvollebregt — APPROVED** [review]
Thanks for these - I'll get them out soon

**Comment by brentvollebregt** [comment]
Thanks for the contribution! This has been released in [v2.38.0](https://github.com/brentvollebregt/auto-py-to-exe/releases/tag/v2.38.0) 🚀

**Pull request merged**